src/applications/appeals/996/containers/ConfirmationPage.jsx

```diff
@@ -36,7 +36,8 @@ export const ConfirmationPage = () => {
 
   const { submission, data } = form;
   const issues = data ? getIssuesListItems(data) : [];
-  const fullName = `${name.first} ${name.middle || ''} ${name.last}`;
+  const fullName = `${name.first || ''} ${name.middle || ''} ${name.last ||
```

Comment:
doesn't need to be covered by this PR but seems like a good candidate for refactoring/a shared function 😄

Reply:
I wish this function was in platform utility code somewhere...

Reply:
So, I ended up adding a shared utility function to get the full name & render it.